packages/tinacms/src/cache/node-cache.ts

```diff
@@ -0,0 +1,32 @@
+import shajs from 'sha.js'
```

Comment:
Should we use the inbuilt `crypto` module instead of this? 
e.g. `import { createHash } from 'crypto';`

Reply:
I ran into issues with that but don't recall exactly what. Will try it again and let you know.

Reply:
thought that might've been the case :/

Reply:
Got it to work using require()